crates/cdk-mintd/src/config.rs

```diff
@@ -108,18 +108,6 @@ pub struct FakeWallet {
     pub max_delay_time: u64,
 }
 
-impl Default for FakeWallet {
```

Comment:
Why remove the default implementation?

With it, it's enough to specify
```toml
ln_backend = "fakewallet"
```

Without it, everyone has to make a choice for each of those (and any future new) values.

Reply:
@ok300 Got it! I missed that context. Revert [here](https://github.com/cashubtc/cdk/pull/470/commits/5138f409f81a1aca48d25f42129acf4daa3eb2d9)